### ISSUE TITLE
JavaDoc Link No Longer Valid

### ISSUE THREAD

**Issue by JRSofty** [drewnoakes/metadata-extractor#180]
To better understand how to use your library I would like to look at the JavaDoc. I don't see a link for it here on GitHub, and the link on your site https://drewnoakes.com/code/exif/ points to a google code URL that is no longer valid, and attempting to download the JavaDoc via Maven also is not working.

**Comment by akidee** [comment]
Yes, please provide an updated link to the recent version. The only valid link I could find is http://javadox.com/com.drewnoakes/metadata-extractor/2.6.2/overview-summary.html

**Comment by drewnoakes** [comment]
Yes now that Google Code is defunct, there isn't a readily available source. I should ideally host the JavaDoc on my site and have a systematic way of keeping it up to date by version.

Can you elaborate on how downloading the JavaDoc via Maven is not working? My understanding is that the JavaDoc is included in the deployment to Maven, so if there's something wrong there I'd like to know. I'm not so familiar with Maven.

**Comment by JRSofty** [comment]
Hi!,

I'm guessing your own build environment for the library isn't in Maven then and that you are manually uploading the jar file to the maven central repository (or someone is if it isn't you). When you use maven to build the javadoc files are all stored into a jar as well and are uploaded with your actual library jar. I would only assume if you are not using maven then you would have to also package the javadoc and source files into separate jar files to upload to the repository.

To be honest, I'm not so worried if I can't get them through maven, as long as there is a valid internet link to visit. So if you were to just host them here then that would be great for me.

--JR

**Comment by drewnoakes** [comment]
The build and publish is all performed using maven. That's why I'm
confused/concerned it's not working for you.

**Comment by JRSofty** [comment]
Hi,

Ok that's strange. I know that I'm using this dependency information in my pom.xml

`<dependency>
    <groupId>com.drewnoakes</groupId>
    <artifactId>metadata-extractor</artifactId>
    <version>2.9.1</version>
</dependency>
`

My Eclipse IDE is setup to download sources and javadoc if they are available in the repository. When I look into my .m2 repository I only see the library jar, but nothing for sources or javadoc.

**Comment by drewnoakes** [comment]
Do you get javadoc for other dependencies in your .m2 directory?

**Comment by JRSofty** [comment]
Yes without a problem.
